circuit_maintenance_parser/parsers/cogent.py

```diff
@@ -0,0 +1,80 @@
+"""Cogent parser."""
+import logging
+import re
+from typing import Dict
+
+from datetime import datetime
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=too-many-branches
+
+
+class ParserCogent(Html):
+    """Notifications Parser for Cogent notifications."""
+
+    provider_type: str = "cogent"
+
+    # Default values for cogent notifications
+    _default_provider = "cogent"
+    _default_organizer = "support@cogentco.com"
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_br(soup.find_all("br"), data)
+            self.parse_title(soup.find_all("title"), data)
+            return [data]
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_br(self, br_results: ResultSet, data: Dict):
+        """Parse <br> tag."""
+        for br in br_results:
+            div_text = br.parent.text
+            for line in re.split(r"\n", div_text):
+                if line.endswith("Network Maintenance"):
+                    data["summary"] = line
+                elif line.startswith("Dear"):
+                    match = re.search("Dear (.*),", line)
+                    if match:
+                        data["account"] = match.group(1)
+                elif line.startswith("Start time:"):
+                    match = re.search("Start time: (.*) \(local time\) (\d+/\d+/\d+)", line)
+                    if match:
+                        start_str = " ".join(match.groups())
+                        start = datetime.strptime(start_str, "%I:%M %p %d/%m/%Y")
+                        data["start"] = self.dt2ts(start)
```

Comment:
I see that the string being matched here references `(local time)` rather than UTC. Do we need to account for time zones in this parser in some way?

Reply:
it should be UTC

Reply:
Ok, are you saying I need to determine what the local time is and convert to UTC?

Reply:
sadly, I have to say yes :(
It's weird they to not offer a GMT/UTC time... but seeing this notification, I only see taking the location and using `pytz` convert between timezone 😞

Reply:
Added a solution for this. It's quite slow during testing. A few seconds extra, but it's noticeable. I can look for an alternative if that's going to be an issue.

Reply:
I'll need to add the additional libraries to `poetry` too